### ISSUE TITLE
Clone bug(lanet repository is over its data quota.)

### ISSUE THREAD

**Issue by Leonisux** [Vincentqyw/image-matching-webui#23]
Downloading checkpoints/PointModel_v0.pth (34 MB)
Error downloading object: checkpoints/PointModel_v0.pth (17c1adf): Smudge error: Error downloading checkpoints/PointModel_v0.pth (17c1adfc8c22b044a9538019101bae0740a9d054c1b4fecd80b52d642272b9ff): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.

Errors logged to 'C:\Users\Administrator\Project\ORS2SEM\data\实验4\image-match\image-matching-webui\.git\modules\third_party\lanet\lfs\logs\20240315T102550.0545338.log'.
Use `git lfs logs last` to view the log.
error: external filter 'git-lfs filter-process' failed
fatal: checkpoints/PointModel_v0.pth: smudge filter lfs failed
fatal: Unable to checkout '58ec4881f43fa082fbd18a61477224266bdc111c' in submodule path 'third_party/lanet'

**Comment by Vincentqyw** [comment]
Fixed! I have made changes to the submodule 'lanet' and added a model downloader for use with 'lanet'. To download the updated code, please use the following command:

``` bash
git clone --recursive https://github.com/Vincentqyw/image-matching-webui.git
```

**Comment by Leonisux** [comment]
Thank you for your response, but after I deploy it locally, it can't run on my computer. I try to run it with the common.utils.run_matching() with the key is 'sift', and it comes an error below.

[Terminal]
Traceback (most recent call last):
  File "xxx\image-match\image-matching-webui\test.py", line 11, in <module>
    kpts, matches_raw, matches_ransac, nums, conf, geom, wraped_img =utils.run_matching(img0,img1,match_threshold,extract_max_keypoints, keypoint_threshold, key)
  File "xxx\image-match\image-matching-webui\common\utils.py", line 356, in run_matching
    pred0 = extract_features.extract(
  File "xxx\image-match\image-matching-webui\hloc\extract_features.py", line 450, in extract
    pred = model({"image": data["image"]})
  File "xxx\lib\site-packages\torch\nn\modules\module.py", line 1130, in _call_impl
    return forward_call(*input, **kwargs)
  File "xxx\image-match\image-matching-webui\hloc\utils\base_model.py", line 24, in forward
    return self._forward(data)
  File "xxx\image-match\image-matching-webui\hloc\extractors\dog.py", line 77, in _forward
    keypoints, scores, descriptors = self.sift.extract(image_np)
ValueError: not enough values to unpack (expected 3, got 2)

I haven't try other keys yet, but it seems like that the return value 'score' doesn't exist according to the Readme on [https://github.com/colmap/pycolmap](url)

**Comment by Vincentqyw** [comment]
Fixed! Please check this commit: https://github.com/Vincentqyw/image-matching-webui/commit/3e8b7386644d85ebae7e090ff86fee65b8a44b6d

**Comment by Leonisux** [comment]
> Fixed! Please check this commit: [3e8b738](https://github.com/Vincentqyw/image-matching-webui/commit/3e8b7386644d85ebae7e090ff86fee65b8a44b6d)

Thank you!

**Comment by Vincentqyw** [comment]
> It's an issue of large model files in Lanet . Download the model here: https://huggingface.co/spaces/Realcat/image-matching-webui/tree/main/third_party/lanet/checkpoints and place it in the right location.

Update: all checkpoints are stored in `huggingface datasets`: https://huggingface.co/Realcat/imcui_checkpoints
